cluster/proto/api/message.proto

```diff
@@ -83,7 +83,8 @@ message QueryResponse {
 }
 
 message AddTenantsRequest {
-  repeated Tenant tenants = 1;
+  repeated string cluster_nodes = 1;
+  repeated Tenant tenants = 2;
```

Comment:
**suggestion (non-blocking):**  can we use [map](https://protobuf.dev/programming-guides/encoding/#maps) 💭

Reply:
Discussed live, we can't use it because the leader needs to make the associate call and the current behaviour is similar. We have an improvement task tracked for future longer term improvements here.